tests/keyboard_test.go

```diff
@@ -81,6 +82,57 @@ func TestKeyboardPress(t *testing.T) {
 		require.Equal(t, "+=@6AbC", v)
 	})
 
+	t.Run("control_or_meta", func(t *testing.T) {
+		t.Skip("FIXME") // See https://github.com/grafana/xk6-browser/issues/424
```

Comment:
Since this PR's goal is not to test combo keys, can we instead add a test that verifies `ControlOrMeta`? For example, pressing `ControlOrMeta` and the mouse double-clicking, and then the backspace key will remove the text from the input box. Another alternative is to replicate the test in the PR description.

Reply:
Good idea! I've changed it to your suggestion by working with `dblClick` 🙂 cd94b7aee2b5a5fb247ca4aec3a615c7c230b457

Reply:
Wait, that works even without `ControlOrMeta` 🤔

Reply:
Ah, my bad. `dblClick` already selects all the text. So, is it better to work out the test in the PR description?

Reply:
Yeah, that definitely works as expected. Here's the change 75aac407096b89ac8e890a2e89b59285e5ac0a35